### PR TITLE
Bug 1205015 - automated l10n screenshots

### PR DESCRIPTION
**Pull request by fluffyemily**
https://bugzilla.mozilla.org/show_bug.cgi?id=1205015

Create fastlane for running l10n builds.

Takes list of devices and languages as arguments

### CODE REVIEW

**Comment by sleroux** [comment]
Looks good! Could we also create a lane specific to marketing as well that's configured to run with all the correct devices/locales? Not part of this PR but maybe a follow up.

File: fastlane/BaseFastfile [hunk 1]
```diff
@@ -203,6 +203,23 @@ platform :ios do
     frameit(white: true)
   end
 
+  desc "Takes localization snapshots \
+  :devices - a list of devices to take snapshots on \
+  :languages - a list of languages to take snapshots of \
+  :output_directory - the directory to output the snapshots to"
+  lane :snapshotL10n do |options|
+    devices = options[:devices].split(",") || ["iPhone 6"]
```
Comment (sleroux) [inline fastlane/BaseFastfile]: Gave it another try this morning with a fresh repo clone and I'm getting the following error when providing no arguments:

`undefined method`split' for nil:NilClass`

I think we need to assign options[:devices]/options[:languages] the defaults prior to splitting.

**Comment by sleroux** [comment]
Ran it with the default options, simulator was able to startup and do its thing but then got:

`./fastlane/screenshots/screenshots.html`
Gist: https://gist.github.com/sleroux/2c8b17bf479cb484b9cd

Should this file be created before hand?

**Comment by fluffyemily** [comment]
Yeah, I guess I've always run it after running another build (l10n, aurora
etc) and those create the required directories. Will take a look.

**Comment by fluffyemily** [comment]
@sleroux I've updated this and rebased it. Should work with no errors now.

**Pull request merged**